src/libs/libchdr/lzma/CpuArch.h

```diff
@@ -140,6 +148,7 @@ MY_CPU_LE_UNALIGN means that CPU is LITTLE ENDIAN and CPU supports unaligned mem
     || defined(__MIPSEL__) \
     || defined(__MIPSEL) \
     || defined(_MIPSEL) \
+    || defined(MY_CPU_LOONGARCH64) \
```

Comment:
No need to test this either, the last check is generic enough to catch LoongArch.